app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user, fereign_key: true
```

Comment:
There is a typing error here. It is actually foreign_key.

Reply:
Thanks for helping peter, We are fixing this issue and will submit the project